main.go

```diff
@@ -22,7 +22,12 @@ var opts struct {
 func main() {
 	_, err := flags.Parse(&opts)
 	if err != nil {
-		fmt.Fprintf(os.Stderr, err.Error())
+		ferr, ok := err.(*flags.Error)
+		
+		if !ok || (ok && ferr.Type != flags.ErrHelp) {
```

Comment:
looks like by default, the flags package prints these kinds of errors to stderr, so we can remove the print block completely and just os.Exit(2)

Reply:
Ok, sounds great i will just do some manual tests with other parse error and commit with the suggested change.

Reply:
unfortunally the flag package don't print this kind of errors in stderr.

What happen is that it prints only to stdout when err == flags.ErrHelp.

So if we print to stderr eachtime when re receive a parse error, it print twice for help.

it is ugly but look this: https://github.com/jessevdk/go-flags/blob/master/examples/main.go#L73